src/SALib/analyze/hdmr.py

```diff
@@ -671,30 +671,38 @@ def f_test(Y, f0, Y_em, R, alpha, m1, m2, m3, n1, n2, n3, n):
     return select
 
 
-def ancova(Y, Y_em, V_Y, R, n):
-    """Analysis of Covariance."""
+def ancova(Y, Y_em, V_Y, n, R):
+    """
+    Perform Analysis of Covariance (ANCOVA) on model and emulator outputs.
+
+    Returns:
+    tuple: A tuple containing sensitivity indices (S),
+            of structural contributions (S_a),
+            and of correlative contributions (S_b).
```

Comment:
Standardize docstring format

```suggestion
    Returns
    --------
    tuple : A tuple containing sensitivity indices (S),
              of structural contributions (S_a) and of correlative contributions (S_b).
```